include/misc/string_error.h

```diff
@@ -9,7 +9,7 @@ namespace dune {
 
 inline std::string string_error(int errnum) {
 #if defined(HAVE_STRERROR_S) || defined(HAVE_STRERROR_R) || defined(HAVE_GNU_STRERROR_R)
-    std::array<char, 256> buffer;
+    std::array<char, 256> buffer{'\0'};
```

Comment:
Using `std::array<char, 256> buffer{};` instead should prevent ReSharper from complaining about it.